pulp_ansible/tests/functional/api/collection/v3/test_sync.py

```diff
@@ -1,196 +1,139 @@
             include_pulp_auth=True,
         )
-        second_remote = self.remote_collection_api.create(second_body)
-        self.addCleanup(self.remote_collection_api.delete, second_remote.pulp_href)
-
-        second_repo = self._create_repo_and_sync_with_remote(second_remote)
-
-        first_content = self.cv_api.list(
-            repository_version=f"{self.first_repo.pulp_href}versions/1/"
+        repository = ansible_repository_factory(remote=remote.pulp_href)
+        result = ansible_bindings.RepositoriesAnsibleApi.sync(
+            repository.pulp_href, {"mirror": mirror}
         )
-        assert len(first_content.results) >= 1
-        second_content = self.cv_api.list(repository_version=f"{second_repo.pulp_href}versions/1/")
-        assert len(second_content.results) >= 1
-
-    def test_sync_collections_from_pulp_using_mirror_second_time(self):
-        """Test sync collections from pulp server using a mirror option the second time."""
-        body = gen_ansible_remote(
-            url="https://galaxy.ansible.com",
-            requirements_file=REQUIREMENTS_FILE,
-            sync_dependencies=False,
-        )
-        remote = self.remote_collection_api.create(body)
-        self.addCleanup(self.remote_collection_api.delete, remote.pulp_href)
-
-        first_repo = self._create_repo_and_sync_with_remote(remote)
-        distribution = self._create_distribution_from_repo(first_repo)
-
-        second_body = gen_ansible_remote(url=distribution.client_url, include_pulp_auth=True)
-        second_remote = self.remote_collection_api.create(second_body)
-        self.addCleanup(self.remote_collection_api.delete, second_remote.pulp_href)
-
-        second_repo = self._create_repo_and_sync_with_remote(second_remote)
+        monitor_task(result.task)
 
-        first_content = self.cv_api.list(repository_version=f"{first_repo.pulp_href}versions/1/")
-        assert len(first_content.results) >= 1
-        second_content = self.cv_api.list(repository_version=f"{second_repo.pulp_href}versions/1/")
-        assert len(second_content.results) >= 1
-
-    def test_sync_collection_named_api(self):
-        """Test sync collections from pulp server."""
-        body = gen_ansible_remote(
-            url="https://galaxy.ansible.com",
-            requirements_file="collections:\n  - rswaf.api",
-            sync_dependencies=False,
+        content = ansible_bindings.ContentCollectionVersionsApi.list(
+            repository_version=f"{repository.pulp_href}versions/1/"
         )
-        remote = self.remote_collection_api.create(body)
-        self.addCleanup(self.remote_collection_api.delete, remote.pulp_href)
-
-        repo = self._create_repo_and_sync_with_remote(remote)
-        distribution = self._create_distribution_from_repo(repo)
-
-        collection = self.collections_v3api.read("api", "rswaf", distribution.base_path)
-
-        assert "api" == collection.name
-        assert "rswaf" == collection.namespace
```

Comment:
The assertion from `test_sync_collection_named_api` test were dropped. Was that intentional?

Reply:
I'm having a hard time even understanding what the tests are supposed to test.
But this one, i think became a parametrised version of another.